### PR TITLE
Fix AsExpression doc link

### PR DESCRIPTION
**Pull request by Ten0**

### CODE REVIEW

File: diesel/src/serialize.rs [hunk 1]
```diff
@@ -186,7 +186,8 @@ where
 /// database, you should use `i32::to_sql(x, out)` instead of writing to `out`
 /// yourself.)
 ///
-/// Any types which implement this trait should also [`#[derive(AsExpression)]`].
+/// Any types which implement this trait should also
+/// [`#[derive(AsExpression)]`](crate::AsExpression).
```
Comment (weiznich) [inline diesel/src/serialize.rs:190]:
```suggestion
/// [`#[derive(AsExpression)]`](derive@crate::AsExpression).
```

Just to be sure that does not "break" again

**Comment by weiznich** [comment]
I'm not sure why the CI is failing here, probably because my suggestion was broken? 🙈

**Comment by Ten0** [comment]
No mine was broken too, I was hoping yours would fix it ^^'
Maybe that's a feature problem or something that prevents it from resolving this derive? Perhaps I should just leave the link as a standard link (the issue came from the `;` I believe).

**Comment by weiznich** [comment]
I've pushed a fixed version. Seems like just the used path was not correct :shrug:

**Comment by Ten0** [comment]
LGTM 🙂

**Pull request merged**